tests/data/dsl-semantic-validation-cases.yaml

```diff
@@ -1184,6 +1184,15 @@
         end: 1
       metadata:
         errorType: schema-version-required
+    - msg: invalid schema undefined
```

Comment:
These errors are now thrown when validating a modular model now that the early throw has been removed